public/static/js/projects.js

```diff
@@ -263,7 +261,7 @@ mciModule.controller('ProjectCtrl', function($scope, $window, $http, $location,
         }
         $scope.projectVars = data.ProjectVars.vars || {};
         $scope.privateVars = data.ProjectVars.private_vars || {};
-        $scope.githubHookID = data.github_hook.hook_id || 0;
```

Comment:
We don't need to know what the hook ID is on the client side if all we care about is that there is one.